cleanlab/classification.py

```diff
@@ -355,15 +383,41 @@ class 0, 1, ..., K-1. `pred_probs` should have been computed using 3 (or
                 clf_kwargs=clf_kwargs,
             )
         elif self.verbose:
-            print("Using provided label_issues_mask instead of finding label issues.")
+            print("Using provided label_issues instead of finding label issues.")
+
+        # label_issues always overwrites self.label_issues_df. Ensure it is properly formatted:
+        if isinstance(label_issues, pd.DataFrame):
+            if "is_label_issue" not in label_issues.columns:
+                raise ValueError("DataFrame label_issues must contain column: 'is_label_issue'")
+            if len(label_issues) != len(labels):
+                raise ValueError("label_issues and labels must have same length")
+            self.label_issues_df = label_issues
+        elif isinstance(label_issues, np.ndarray):
+            if label_issues.shape != labels.shape:
+                raise ValueError(
+                    "If it is numpy.array, label_issues must have same shape as labels"
+                )
+            if not ((label_issues == 0) | (label_issues == 1)).all():
```

Comment:
Why not check `label_issues.dtype` instead?

Reply:
updated to:

```
if not label_issues.dtype is np.dtype('bool'):
                raise ValueError(
                    "If it is numpy.array, label_issues.dtype must be 'bool'."
                )
```